infra/equaler.go

```diff
@@ -1,4 +1,4 @@
-package models
+package infra
```

Comment:
Golang common lore discourages use of generic words like "utils" or in this case "infra" for packages. What is this package about? If it's about gorm support, let's call it "gormsupport". Go packages are usually smaller than Java packages. Don't hesitate to make many of them instead of lumping too many things in the same package